src/plugins/intel_gpu/tests/unit/test_cases/slice_gpu_test.cpp

```diff
@@ -163,17 +308,26 @@ TYPED_TEST(SliceTest, bfzyx) {
     params.input = this->template AllocateTensor<TypeParam>(
         input_shape, format::bfzyx, helpers::GenInput<TypeParam>(input_shape));
     params.start = this->template AllocateTensor<int64_t>(
-        ov::PartialShape{ 5, 1, 1, 1 }, format::bfzyx, { 0, 0, 0, 0, 0 });
+        ov::PartialShape{ 5 }, format::bfzyx, { 0, 0, 0, 0, 0 });
     params.stop = this->template AllocateTensor<int64_t>(
-        ov::PartialShape{ 5, 1, 1, 1 }, format::bfzyx, { 1, 2, 2, 2, 2 });
+        ov::PartialShape{ 5 }, format::bfzyx, { 1, 2, 2, 2, 2 });
     params.step = this->template AllocateTensor<int64_t>(
-        ov::PartialShape{ 5, 1, 1, 1 }, format::bfzyx, { 1, 1, 1, 1, 1 });
+        ov::PartialShape{ 5 }, format::bfzyx, { 1, 1, 1, 1, 1 });
     params.wanted_output = this->template AllocateTensor<TypeParam>(
         ov::PartialShape{ 1, 2, 2, 2, 2 }, format::bfzyx, { 
             0,   1,   5,   6,   60,  61,  65,  66,
             600, 601, 605, 606, 660, 661, 665, 666
         });
 
+    params.is_caching_test = true;
+
+    this->RunAllTestCasesForParams(params);
+    params.is_input_dynamic = true;
+    params.is_start_dynamic = true;
+    params.is_step_dynamic = true;
+    params.is_stop_dynamic = true;
+    params.is_axes_dynamic = true;
+
     this->RunAllTestCasesForParams(params);
 }
```

Comment:
Could you please add functional tests as well? They are located here `src/plugins/intel_gpu/tests/functional/single_layer_tests/dynamic/`
And currently `StridedSlice` operation replaces `Slice` op during `SliceToStridedSlice` transformation, so you need to disable this transformation to keep slice op in ov::Model

Reply:
1. Ok, will add some functional tests as well.
2. Disabling SliceToStridedSlice transformation should be done in separate mr